gapis/api/vulkan/vulkan.go

```diff
@@ -29,9 +29,11 @@ import (
 )
 
 type CustomState struct {
-	SubcommandIndex   sync.SubcommandIndex
+	SubcommandIndex   api.SubcommandIndex
 	CurrentSubmission *api.Cmd
 	HandleSubcommand  func(interface{}) `nobox:"true"`
+	AddCommand        func(interface{}) `nobox:"true"`
```

Comment:
That was a hold-over from when we added things differently to the custom state, I think I can remove (will make sure)